parkhangs-expressapp/controllers/userController.js

```diff
@@ -102,6 +103,21 @@ const updateUser = async (req, res) => {
             user.savedParks = body.savedParks
         }
 
+        if (body.savedEvents !== undefined) {
+            try {
+                await saveEvents(user.savedEvents, body.savedEvents)
+            } catch (error) {
+                console.error(error)
+                console.log('unable to update event favorite count')
+                return res.status(404).json({
```

Comment:
remove the 2 console statements

Reply:
again, necessary for error debugging and common practice.